scipy/stats/_bootstrap.py

```diff
@@ -373,7 +388,7 @@ def bootstrap(data, statistic, *, axis=0, confidence_level=0.95,
     # Calculate percentile interval
     alpha = (1 - confidence_level)/2
     if method == 'bca':
-        interval = _bca_interval(data, statistic, axis, alpha, theta_hat_b)
+        interval = _bca_interval(data, statistic, -1, alpha, theta_hat_b)
```

Comment:
This needs to make it into 1.7.0 whether or not `paired` is ready.

Reply:
But then I suggest you put these 2 small changes in a PR so we merge it now. As the branching is very soon.

cc @tylerjereddy

Reply:
Right, and I'll do that if need be, but I figured I'd try to add them in these PRs that were asked to be added before branching.

Reply:
This will already be fixed when I resolve the merge conflict we'll get from gh-14107.